src/main/views/privacy-policy.njk

```diff
@@ -0,0 +1,125 @@
+{% extends "layout.njk" %}
+{% from "support.njk" import supportWidget %}
+{% set heading = t('Privacy policy') %}
+
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <p>{{ t('When you use the money claims service we will ask you for some personal data.' +
```

Comment:
No whitespace at the end of line. Please add one.

Reply:
👍